admin/ucf-academic-calendar-config.php

```diff
@@ -227,6 +239,24 @@ public static function settings_init() {
 
 			register_setting(
 				'ucf_acad_cal',
+				self::$option_prefix . 'is_important'
+			);
+
+			add_settings_field(
+				self::$option_prefix . 'is_important',
+				'Pull Important Events',
+				array( 'UCF_Acad_Cal_Config', 'display_settings_field' ),
+				'ucf_acad_cal',
+				'ucf_acad_cal_general',
+				array(
+					'label_for'   => self::$option_prefix . 'is_important',
+					'description' => 'When checked, only events marked as "isImportant" will be pulled from the feed.',
```

Comment:
Update isImportant to is_important in the description.

Reply:
The field in the json is `isImportant`:

```
"category": "Academic Dates and Deadlines",
"url": null,
"summerSession": null,
"isImportant": false,
"uniqueTag": "",
"dtstart": "2017-11-03 00:00:00Z",
"dtend": "",
```

Reply:
ah